packages/twenty-server/src/metadata/object-metadata/types/index-metadata.ts

```diff
@@ -0,0 +1,30 @@
+export type IndexType = 'btree' | 'gin';
+
+export type IndexMetadata = {
+  // Name of the index, can be delegated to the engine.
+  name?: string;
```

Comment:
@charlesBochet I can remove all of that stuff, later in the code I do Pick<IndexMetadata, 'columns' | 'name'> so the rest is ignored. This is for @FelixMalfait 😄